envs/conda_build_config.yaml

```diff
@@ -225,7 +225,8 @@ pathy:
   - 0.6.*
 pip:
   - 21.*
-pkg_config:
+#using pkg_config is not working here with ffmpeg-feedstock, so updated to pkgconfig
+pkgconfig:
```

Comment:
if pkg_config is present in this configfile then ffmpeg build fails. So we have to update the pinname to pkgconfig. We are not updating the pin here, just the pin tag which is causing the issue. So need to update other affected packages as well.

Reply:
if pkg_config is present in this configfile then ffmpeg build fails. So we have to update the pinname to pkgconfig. We are not updating the pin here, just the pin tag which is causing the issue. So need to update other affected packages as well.

Reply:
The other option is to remove _pkg_config_ pin altogether because v0.29.2 is the only version available on Anaconda as of now, but that will also affect the recipes that are presently using _pkg_config_. So presently keeping the pin and updating the other packages. Thankfully the affected packages are few.